src/System.IO.FileSystem/tests/File/Delete.cs

```diff
@@ -122,6 +124,30 @@ public void Unix_NonExistentPath_Nop()
         }
 
         [Fact]
+        [OuterLoop("Needs sudo access")]
+        [PlatformSpecific(TestPlatforms.Linux)]
+        [Trait(XunitConstants.Category, XunitConstants.RequiresElevation)]
+        public void Unix_NonExistentPath_ReadOnlyVolume()
+        {
+            ReadOnly_FileSystemHelper(readOnlyDirectory =>
+            {
+                Delete(Path.Combine(readOnlyDirectory, "DoesNotExist"));
+            });
+        }
+
+        [Fact]
+        [OuterLoop("Needs sudo access")]
```

Comment:
Trait doesn't work as far as we can tell- there might be a filter in the CI scripts. I'm looking into how to tell we're elevated in Unix- we only have Windows checks for this afaik.